cluster/addons/fluentd-gcp/fluentd-gcp-ds.yaml

```diff
@@ -61,7 +61,7 @@ spec:
               then
                 exit 1;
               fi;
-              LAST_MODIFIED_DATE=`stat /var/log/fluentd-buffers | grep Modify | sed -r "s/Modify: (.*)/\1/"`;
+              LAST_MODIFIED_DATE=`find /var/log/fluentd-buffers -type f -exec stat \{} --printf="%y\n" \; | sort -n -r | head -n 1`;
```

Comment:
Executing the `stat` command for every file is probably quite inefficient. How about using `xargs` instead:
```sh
LAST_MODIFIED_DATE=`find /var/log/fluentd-buffers -type f | xargs -- stat --format="%y" | sort -rn | head -n 1`
```
?

Reply:
```
touch -d "@$(date -d"${LIVENESS_THRESHOLD_SECONDS} seconds ago" '+%s')" /tmp/marker
find /var/log/fluentd-buffers -type f -newer /tmp/marker
```

Reply:
Updated, PTAL.